GoogleForCreators/web-stories-wp
# Editor: Remove duplicate icons

## Summary

- Removing duplicate icons that were in the story editor.
- Move unique icons to the design system.

## Relevant Technical Choices

- checkmark in design system is thinner than the old one, so tried to make them look as similar as possible.
- the 'more' dots are smaller in the design system. The new 'more' button has a smaller icon, but that can be increased if we want it to be.
- `KeyArrowUp` can be removed once https://github.com/google/web-stories-wp/pull/8800 is merged

## To-do

- [x] Remove `KeyArrowUp` once #8800 is merged

## User-facing changes

||Before|After|
|--|--|--|
|Design Panel - Advanced Dropdown (Magnifier, Close, and Checkmark)|![image](https://user-images.githubusercontent.com/22185279/131394295-e4145542-6271-4115-b15d-c0f2ff64b5e9.png)|![image](https://user-images.githubusercontent.com/22185279/131395223-13004551-ad3a-4842-8fca-6ccc07a6de5c.png)|
|Media Panel - More button (Dots)|![image](https://user-images.githubusercontent.com/22185279/131395795-c1a09e18-f435-434e-b1ee-2e9bc0cc4c5e.png) ![more-before](https://user-images.githubusercontent.com/22185279/131400287-497cacda-ddfc-468e-89fd-b877fa73f029.gif)|![image](https://user-images.githubusercontent.com/22185279/131396810-d74fe85c-ffbc-4c35-869d-bb7f3e19d7f1.png) ![more-after](https://user-images.githubusercontent.com/22185279/131400309-85971715-003d-4feb-ad4e-bbcd80c16f60.gif)|
|Media 3P Panel - Volume Off Icon (Muted)<br/><br/>These are identical because the icon was moved to the design system|![image](https://user-images.githubusercontent.com/22185279/131401343-8121765e-9ce3-47ba-baf7-4fc28d422a15.png)|![image](https://user-images.githubusercontent.com/22185279/131401300-5fd949ae-a088-41d5-a5c6-259de0428252.png)|
|Design Panel - Union Icon<br/><br/>These are identical because the icon was moved to the design system|![image](https://user-images.githubusercontent.com/22185279/131402484-3c785dac-fe05-45fd-8900-890cca98ea23.png)|![image](https://user-images.githubusercontent.com/22185279/131402424-ca9109be-6708-4c5d-8a08-b46cc4d5fd84.png)
|Design Panel - Turning Line Icon<br/><br/>These are identical because the icon was moved to the design system|![image](https://user-images.githubusercontent.com/22185279/131403020-82075d77-cf11-46f3-8bff-21658b7e7059.png)|![image](https://user-images.githubusercontent.com/22185279/131403101-fa3df6a0-6efb-4b28-bf4a-98efd6acd5d0.png)|


## Testing Instructions

A little tedious to test, but I'll try to make it easy. 




This PR can be tested by following these steps:

|Icon|Location|
|--|--|
|ArrowDown Icon|This icon wasn't used, so it was removed. Nothing to check.|
|Checkmark Icon|This can be found in the `AdvancedDropDown` component.<br/>1. Add a text element to the canvas.<br/>2. Go to the font panel in the inspector.<br/>3. The font picker is an advanced dropdown - click the font picker.<br/>4. Type in text or scroll to find the selected option. The checkmark should be visible there.|
|Close Icon|This can be found in the `AdvancedDropDown` component.<br/>1. Add a text element to the canvas.<br/>2. Go to the font panel in the inspector.<br/>3. The font picker is an advanced dropdown - click the font picker.<br/>4. Type in text in the textbox to see the close icon appear.|
|Dropdown Icon|This icon wasn't used, so it was removed. Nothing to check.|
|Search Icon|This can be found in the `AdvancedDropDown` component.<br/>1. Add a text element to the canvas.<br/>2. Go to the font panel in the inspector.<br/>3. The font picker is an advanced dropdown - click the font picker.<br/>4. Should see the magnifier on the left side of the textbox.|
|Media Icon|This icon wasn't used, so it was removed. Nothing to check.|
|More Icon|This can be found in the Media Panel.<br/>1. Hover a piece of media in the media panel.<br/>2. Should see the more button appear in the top right of the picture.|
|Union Icon|This icon can be found in the design panel.<br/>1. Add a piece of text.<br/>2. Click on the lock icon in the Corner Radius panel. Should see the union icon.|
|TurningLine Icon|This icon can be found in the design panel.<br/>1. Add a piece of text.<br/>2. Click on the lock icon in the Text Box panel. Should see the turning line icons.|

## Reviews

### Does this PR have a security-related impact?

no

### Does this PR change what data or activity we track or use?

no

### Does this PR have a legal-related impact?

no

## Checklist




---



Fixes #8555
